client/jetpack-connect/connection-notice-types.js

```diff
@@ -0,0 +1,26 @@
+/** @format */
+
+/**
+ * Shared notice types
+ *
+ * These notice types are indicators of Jetpack Connection status.
+ */
+
+export const ALREADY_CONNECTED = 'alreadyConnected';
+export const ALREADY_CONNECTED_BY_OTHER_USER = 'alreadyConnectedByOtherUser';
+export const ALREADY_OWNED = 'alreadyOwned';
+export const DEFAULT_AUTHORIZE_ERROR = 'defaultAuthorizeError';
+export const IS_DOT_COM = 'isDotCom';
+export const JETPACK_IS_DISCONNECTED = 'jetpackIsDisconnected';
+export const JETPACK_IS_VALID = 'jetpackIsValid';
+export const NOT_ACTIVE_JETPACK = 'notActiveJetpack';
+export const NOT_CONNECTED_JETPACK = 'notConnectedJetpack';
+export const NOT_EXISTS = 'notExists';
+export const NOT_JETPACK = 'notJetpack';
+export const NOT_WORDPRESS = 'notWordPress';
+export const OUTDATED_JETPACK = 'outdatedJetpack';
+export const RETRY_AUTH = 'retryAuth';
+export const RETRYING_AUTH = 'retryingAuth';
+export const SECRET_EXPIRED = 'secretExpired';
+export const USER_IS_ALREADY_CONNECTED_TO_SITE = 'userIsAlreadyConnectedToSite';
+export const WORDPRESS_DOT_COM = 'wordpress.com';
```

Comment:
I put this into it's own module because I expected to `import * as noticeTypes from '…'` but ended up never doing that. I think it's fine here, but could go into the `constants` module as well.

Reply:
Totally fine like this.